katas/content/multi_qubit_gates/preparing_bell_state/index.md

```diff
@@ -1,3 +1,3 @@
 **Input:** Two qubits in state $|00\rangle$, stored in an array of length 2.
 
-**Goal:** Transform the system into the Bell state $\ket{\Phi^+} = \frac{1}{\sqrt{2}}\big(|00\rangle + |11\rangle\big)$.
```

Comment:
What was the change supposed to be here? There was a conflict with my fix to add \ket to \Phi, and I couldn't figure out what was the proposed change

Reply:
The only change is a newline added to the end of the file.